src/main/java/org/mutabilitydetector/checkers/AllChecksRunner.java

```diff
@@ -62,6 +62,7 @@ public AnalysisResult runCheckers(ImmutableList<AnalysisResult> knownResultsSoFa
         Map<IsImmutable, Integer> results = newHashMap();
         Collection<MutableReasonDetail> reasons = newArrayList();
         Collection<AnalysisError> errors = newArrayList();
+        boolean isClassSelfReferenced=false;
```

Comment:
Here and elsewhere: try to follow the existing code style with respect to whitespace: spaces around `=`, space after comma, space after `if`, etc.

Reply:
Yep, done